woocommerce/woocommerce-android
# Update configuration

This PR updates the CI configuration in order to fix an issue with uploading symbols to Sentry.

### To test:
- Verify that the CI is green and the Installable build job run without errors.

Update release notes:
